pkg/kubectl/cmd/logs/logs.go

```diff
@@ -151,6 +158,7 @@ func NewCmdLogs(f cmdutil.Factory, streams genericclioptions.IOStreams) *cobra.C
 	cmd.Flags().StringVarP(&o.Container, "container", "c", o.Container, "Print the logs of this container")
 	cmdutil.AddPodRunningTimeoutFlag(cmd, defaultPodLogsTimeout)
 	cmd.Flags().StringVarP(&o.Selector, "selector", "l", o.Selector, "Selector (label query) to filter on.")
+	cmd.Flags().IntVar(&o.MaxFollowConcurency, "max-follow-concurency", o.MaxFollowConcurency, "Specify maximum number of concurrent logs to follow when using by a selector. Defaults to 5.")
```

Comment:
I'd propose naming this `max-requests`

Reply:
Maximum number of parallel log requests when using selector. Defaults to 5.

Reply:
We could settle for a name in the middle, maybe something like `max-log-requests`

Reply:
sgtm

Reply:
@soltysh @juanvallejo I'm not sure that about `max-log-requests`: sound  too vague to me. When you do something like `kubectl logs -l deployment/some-name`, it will basically create N requests, but kubectl will be reading them sequentially.

Probably it's ok for end-users, because they don't care about the number of sequentially requests, right?